public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.js

```diff
@@ -57,11 +73,15 @@ class SpeakersBlockContent extends Component {
 		}
 
 		return (
+			// this feels a bit overwhelming. i think this would be easier to grok and maintain if it were modularized more
+			// maybe pull out the `wordcamp-speaker-content` and session parts into functions that just get called from here?
+
 			<ul className={ classnames( containerClasses ) }>
 				{ speakerPosts.map( ( post, i ) =>
 					<li
 						key={ i }
 						className={ classnames( 'wordcamp-speaker', 'wordcamp-speaker-' + post.slug ) }
+						// should we use decodeEntities here too?
```

Comment:
Maybe? Unless the `classnames` function takes care of that. I'll have to doublecheck the source...

Reply:
Nope, doesn't look like `classnames` does any sanitization. So yes, we should probably use `decodeEntities` here.